discover/index.html

```diff
@@ -75,15 +74,13 @@ <h3 class="text-left" id="services"><i class="far fa-fw fa-map"></i> Available S
   <li><i class="fas fa-fw fa-unlock"></i> <code>NAIP {year} RGB</code> – 1-meter statewide natural-color aerial photography
     <ul class="dotless">
       <li><em>Available Years: 2006, 2009, 2011, 2014, 2016</em>
-    </ul>
   <li><i class="fas fa-fw fa-unlock"></i> <code>NAIP {year} NRG</code> – 1-meter statewide color infrared aerial photography
     <ul class="dotless">
       <li><em>Available Years: 2006, 2011, 2014, 2016</em>
-    </ul>
   <li><i class="fas fa-fw fa-unlock"></i> <code>Hillshade</code> – 10-meter statewide combined hillshade and slope
 </ul>
 <h3 class="text-left"><i class="far fa-fw fa-thumbs-up"></i> Best Practices</h3>
 <ul>
-<li>It is strongly suggested that you use <code>WMTS</code> for the best draw time/perceived performance.
+<li>It is strongly recommended that you use <code>WMTS</code> for the best draw time/perceived performance.
```

Comment:
Here I thought "recommend" read more smoothly. But that's a little subjective, so if you disagree, feel free to change it back!